### ISSUE TITLE
Not enough jazz (adding some music at least in jazz_bar)

### ISSUE THREAD

**Issue by VinceYZ** [Foohy/jazztronauts#80]
It's really quiet, and while it may be good while exploring maps and finding shards, it really doesn't make much sense in hub worlds like jazz_bar. It would be really cool to have some music flowing

**Comment by eros71-dev** [comment]
I agree, some background music for the bar would be nice, it feels a little too silent.
